install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1156,133 +1155,52 @@ deployed. For example, if the project is named *logging*:
-for the deployer.
-+
-----
-$ oc import-image <name>:<version> --from <prefix><name>:<tag>
-----
-+
-For example:
+. Generate any missing service accounts and roles:
 +
 ----
-$ oc import-image logging-auth-proxy:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-auth-proxy:3.2.1
-$ oc import-image logging-kibana:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-kibana:3.2.1
-$ oc import-image logging-elasticsearch:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-elasticsearch:3.2.1
-$ oc import-image logging-fluentd:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-fluentd:3.2.1
+$ oc process logging-deployer-account-template | oc apply -f -
 ----
 
-endif::openshift-enterprise[]
-
-. Next, scale Elasticsearch back up incrementally so that the cluster has time
-to rebuild.
-
-.. To begin, scale up to 1:
+. In preparation for running the deployer, ensure that you have the configurations
+for your current deployment in the xref:../aggregate_logging.adoc#specifying-deployer-parameters[logging-deployer configmap].
 +
-----
-$ oc scale dc/logging-es-<unique_name> --replicas=1
-----
-+
-Follow the logs of the resulting pod to ensure that it is able to recover its
-indices correctly and that there are no errors:
+[IMPORTANT]
+====
+Ensure that your image version is the latest version, not the currently installed
+version.
+====
+
+. Run the deployer with the parameter in `upgrade` mode:
 +
 ----
-$ oc logs -f <pod_name>
+$ oc new-app logging-deployer-template -p MODE=upgrade
 ----
 +
-If that is successful, you can then do the same for the operations cluster, if
-one was previously used.
-
-.. After all Elasticsearch nodes have recovered their indices, continue to scale it
-back up to the size it was prior to doing maintenance. Check the logs of the
-Elasticsearch members to verify that they have correctly joined the cluster and
-recovered.
-
-. Now scale Kibana and Fluentd back up to their previous state. Because Fluentd
-was shut down and allowed to push its remaining records to Elasticsearch in the
-previous steps, it can now pick back up from where it left off with no loss of
-logs, provided any unread log files are still available on the node.
-
-. In the latest version, Kibana will display indices differently now in order
-to prevent users from being able to access the logs of previously created
-projects that have been deleted.
-+
-Due to this change, your old logs will not appear automatically. To migrate your
-old indices to the new format, rerun the deployer with `-v MODE=migrate` in addition
-to your prior flags. This should be run while your Elasticsearch cluster is running, as the
-script must connect to it to make changes.
+Running the deployer in this mode handles scaling down the components to minimize
+loss of logs, patching configs, generating missing secrets and keys, and scaling
+the components back up to their previous replica count.  All in a non-destructive
```

Comment:
I was thinking more of destructive in the sense that we don't reinstall how we've suggested before so any DC changes aren't lost... We can probably leave it off though